Sample/RRFPSBar.m

```diff
@@ -59,7 +61,11 @@ - (void)dealloc {
 
 
 - (id)init {
-    if( (self = [super initWithFrame:[[UIApplication sharedApplication] statusBarFrame]]) ){
+        CGRect barFrame = CGRectMake(0,
+                                     self.safeAreaInsets.top,
```

Comment:
It's been so long since I've been in Obj-C land.. we're ok accessing `self` before it's initialized here? / it's guaranteed to have the right value right now?

Reply:
oh, good call. I should fix that. Actually not sure why this was working.